### PR TITLE
update changelog for the 0.3 series

### PR DESCRIPTION
**Pull request by esc**
As title

### CODE REVIEW

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+* Dynamic generation of `scipy.special.signatures` (with thanks to Brandon T.
+  Willard)
+* Github Actions CI
+
+Importantly, the dynamic generation of the signatures makes numba-scipy now
+compatible with multiple SciPy releases simultaneously.
+
+Contributors
+* Brandon T. Willard
+* Stuart Archibald
```
Comment (stuartarchibald) [inline CHANGE_LOG:14]: I don't recall adding a patch?!

File: CHANGE_LOG [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
 
-In development
+* Dynamic generation of `scipy.special.signatures` (with thanks to Brandon T.
+  Willard)
+* Github Actions CI
+
+Importantly, the dynamic generation of the signatures makes numba-scipy now
+compatible with multiple SciPy releases simultaneously.
+
+Contributors
```
Comment (stuartarchibald) [inline CHANGE_LOG:12]: New line needed here to start a list.

File: CHANGE_LOG [hunk 3]
```diff
@@ -1,7 +1,18 @@
 Version 0.3
 -----------
+This release adds:
 
-In development
+* Dynamic generation of `scipy.special.signatures` (with thanks to Brandon T.
```
Comment (stuartarchibald) [inline CHANGE_LOG:5]:
```suggestion
* Dynamic generation of ``scipy.special.signatures`` (with thanks to Brandon T.
```
double backticks else you just get italics.

File: CHANGE_LOG [hunk 4]
```diff
@@ -1,7 +1,18 @@
...
+  Willard)
+* Github Actions CI
+
+Importantly, the dynamic generation of the signatures makes numba-scipy now
```
Comment (stuartarchibald) [inline CHANGE_LOG:9]:
```suggestion
Importantly, the dynamic generation of the signatures now makes ``numba-scipy``
```

**Review by stuartarchibald — COMMENTED** [review]
Thanks for the patch, few minor things to address else looks good.

File: CHANGE_LOG (hunk 1 above)
Comment (esc) [reply CHANGE_LOG:14]: Your name appears in the git-log:

```
 💣 zsh» git --no-pager shortlog -sn 0.3.0dev0..
    24	Valentin Haenel
     7	esc
     4	Brandon T. Willard
     2	stuartarchibald
```

Probably a merge?

Comment (esc) [reply CHANGE_LOG:14]: Yes it was merges:

```
 💣 zsh» git --no-pager shortlog -sn --no-merges 0.3.0dev0..
    24	Valentin Haenel
     4	Brandon T. Willard
     1	esc
```

File: CHANGE_LOG (hunk 4 above)
Comment (esc) [reply CHANGE_LOG:9]: Oh, yes, a  "germanism".

File: CHANGE_LOG (hunk 2 above)
Comment (esc) [reply CHANGE_LOG:12]: fixed

File: CHANGE_LOG (hunk 3 above)
Comment (esc) [reply CHANGE_LOG:5]: fixed

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch.

**Comment by esc** [comment]
@stuartarchibald thanks for the review, release will commence now.

**Pull request merged**